amazon-kinesis-client-multilang/src/test/java/software/amazon/kinesis/multilang/MultiLangDaemonConfigTest.java

```diff
@@ -35,62 +31,168 @@
 import software.amazon.awssdk.auth.credentials.AwsCredentials;
 import software.amazon.awssdk.auth.credentials.AwsCredentialsProvider;
 import software.amazon.kinesis.multilang.config.KinesisClientLibConfigurator;
-import software.amazon.kinesis.multilang.config.MultiLangDaemonConfiguration;
 
 @RunWith(MockitoJUnitRunner.class)
 public class MultiLangDaemonConfigTest {
-    private static String FILENAME = "some.properties";
+    private static final String TEST_FILENAME = "some.properties";
+    private static final String TEST_EXE = "TestExe.exe";
+    private static final String TEST_APPLICATION_NAME = "TestApp";
+    private static final String TEST_STREAM_NAME = "fakeStream";
+    private static final String TEST_STREAM_NAME_IN_ARN = "FAKE_STREAM_NAME";
+    private static final String TEST_REGION = "us-east-1";
+    private static final String TEST_STREAM_ARN = "arn:aws:kinesis:us-east-2:012345678987:stream/" + TEST_STREAM_NAME_IN_ARN;
+
+    @Mock
+    ClassLoader classLoader;
 
     @Mock
     private AwsCredentialsProvider credentialsProvider;
     @Mock
     private AwsCredentials creds;
-    @Mock
+
     private KinesisClientLibConfigurator configurator;
+    private MultiLangDaemonConfig deamonConfig;
+
+    /**
+     * Instantiate a MultiLangDaemonConfig object based on the properties passed in.
+     * @param streamName
+     * @param streamArn
+     * @param regionName
+     * @throws IOException
+     */
+    public void setup(String streamName, String streamArn, String regionName) throws IOException {
```

Comment:
Would you kindly convert `String regionName` to `Region region` (or whatever parameter name you choose)?

Please avoid variants of https://wiki.c2.com/?PrimitiveObsession  Let's keep objects strongly-typed for as long as we can.